srcpkgs/hunspell-ro_RO/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'hunspell-ro_RO'
+pkgname=hunspell-ro_RO
+version=3.3.10
+revision=1
+create_wrksrc=yes
+hostmakedepends="unzip"
+short_desc="Romanian dictionary for hunspell"
+maintainer="Klorophatu <klorophatu@krutt.org>"
+license="GPL-2.0-or-later, LGPL-2.1-or-later, MPL-1.1"
+homepage="https://sourceforge.net/projects/rospell/"
+distfiles="${SOURCEFORGE_SITE}/project/rospell/Romanian%20dictionaries/dict-${version}/ro_RO.${version}.zip"
+checksum=7f128d64ea06c9e6711c30b118c0afeefb014d8f33c92daccdf455aba2d04519
+
+do_install() {
+	vinstall ${wrksrc}/ro_RO.aff 644 /usr/share/hunspell
+	vinstall ${wrksrc}/ro_RO.dic 644 /usr/share/hunspell
+	vlicense ${wrksrc}/README
```

Comment:
I'm not sure `${wrksrc}` should be required here...

Also, use `usr/share/hunspell`.